spikesorters/spyking_circus/config_default.params

```diff
@@ -3,21 +3,22 @@
 ### file format. Otherwise, launch the code and a message will tell you what is needed
 
 [data]
-file_format    = numpy      # Can be raw_binary, openephys, hdf5, ... See >> spyking-circus help -i for more info
+file_format    = raw_binary # Can be raw_binary, openephys, hdf5, ... See >> spyking-circus help -i for more info
 sampling_rate  = {}
 stream_mode    = None       # None by default. Can be multi-files, or anything depending to the file format
 mapping        = {}         # Mapping of the electrode (see http://spyking-circus.rtfd.ord)
 suffix         =            # Suffix to add to generated files
 global_tmp     = True       # should be False if local /tmp/ has enough space (better for clusters)
 overwrite      = True       # Filter or remove artefacts on site (if write access is possible). Data are duplicated otherwise
 parallel_hdf5  = True       # Use the parallel HDF5 feature (if available)
+data_offset    = 0
+data_dtype     = float32
```

Comment:
raw data is not always float32.
You have to template this parameters depending of recodring.get_dtype()